addons/account/views/account_view.xml

```diff
@@ -811,6 +811,7 @@
                     <field name="type_tax_use"/>
                     <field name="description" string="Short Name"/>
                     <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
+                    <field name="active" invisible="1" groups="base.group_no_one"/>
```

Comment:
@hmo-odoo removed groups and kept only invisible attribute, note that if there is groups attribute then if any developer add attrs on same field in future then attrs will not be applied in any case, check_group method removes attrs